docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,1344 @@
+{
```

Comment:
<p>I don't think you mean "rewrite" but, instead, you mean "rearrange"?</p><p> </p><p>I had a hard time understanding when <code>i = j</code> becomes relevant until below. I think you should just substitute the rearrange Pearson correlation into (*)  first and then remind the reader that <code>\rho = 1 </code>  after the substitution</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
> I don't think you mean "rewrite" but, instead, you mean "rearrange"?

Yes. I think rearrange is the correct wording here! You have eyes of an eagle when you review something as you can see all small details.. Very cool!


> I had a hard time understanding when `i = j` becomes relevant until below. I think you should just substitute the rearrange Pearson correlation into (*) first and then remind the reader that `\rho = 1 ` after the substitution

Yes...it feels things are not connected here. I can use `\rho_i,j` and `\rho_i,i`. And, then replace the latter with 1 followed by a note to the reader.